nsxt/provider.go

```diff
@@ -518,31 +505,9 @@ func Provider() *schema.Provider {
 			"nsxt_policy_gateway_flood_protection_profile_binding":     resourceNsxtPolicyGatewayFloodProtectionProfileBinding(),
 			"nsxt_policy_compute_sub_cluster":                          resourceNsxtPolicyComputeSubCluster(),
 			"nsxt_policy_tier0_inter_vrf_routing":                      resourceNsxtPolicyTier0InterVRFRouting(),
-			"nsxt_vpc_security_policy":                                 resourceNsxtVPCSecurityPolicy(),
-			"nsxt_vpc_group":                                           resourceNsxtVPCGroup(),
-			"nsxt_vpc_gateway_policy":                                  resourceNsxtVPCGatewayPolicy(),
```

Comment:
These 3 were also released in v3.8.0.
If we made significant changes to them it might be the case of removing them.